src/styles/about-us.scss

```diff
@@ -0,0 +1,41 @@
+.about-us {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 104px;
+  color: $darkColor;
+  @media (max-width: $displayLarge) {
+    flex-direction: column;
+    padding-top: 85px;
+  }
+  @media (max-width: $displaySmall) {
+    padding-top: 64px;
+  }
+
+  &__top-title {
+    font-size: $title2LvlFontSize;
+    line-height: $title2LvlLineHeight;
```

Comment:
the same, it's better to create a mixin with some most used combinations of styles

Reply:
No, mixin needs when you need all the time use the same combianation of properties. But in this projects it depents of element. So var is better.

Reply:
ok then rename them

Reply:
why? name says that it is font size of 2lvl title